src/Features/LanguageServer/Protocol/Handler/ILspServiceRequestHandler.cs

```diff
@@ -0,0 +1,22 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.CommonLanguageServerProtocol.Framework;
+using Microsoft.VisualStudio.LanguageServer.Protocol;
+
+namespace Microsoft.CodeAnalysis.LanguageServer.Handler;
+
+internal interface ILspServiceRequestHandler<RequestType, ResponseType> :
```

Comment:
Can you please rename all of the type parameters to start with `T`. Its confusing on line 12 where `RequestType` is a type parameter, but `RequestContext` isn't (right?? I'm not even sure)

Reply:
Fair, I think I got mixed up because [Roslyn is](https://github.com/dotnet/roslyn/blob/main/src/Features/LanguageServer/Protocol/Handler/Formatting/AbstractFormatDocumentHandlerBase.cs#L21) a [bit inconsistent](https://github.com/dotnet/roslyn/blob/20c38a7bf738937a9a50cb2c0e7954b372068b6a/src/EditorFeatures/Core/ExternalAccess/VSTypeScript/Api/AbstractVSTypeScriptRequestHandler.cs#L18). Will centralize as you describe.